src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/BQSRPipelineSpark.java

```diff
@@ -33,33 +33,58 @@
 import java.util.List;
 
 /**
- * BQSR. The final result is analysis-ready reads.
+ * The full BQSR pipeline in one tool to run on Spark.
```

Comment:
Update program group and fix the example command

Reply:
Fixed